includes/admin-reports/enter_attendee_payments.php

```diff
@@ -197,10 +197,10 @@ function enter_attendee_payments() {
 
 			if ( $org_options["use_attendee_pre_approval"] == "Y" ) {
 			
-				$pre_approve = sanitize_text_field($_POST['pre_approve']);
+				$pre_approve = int()$_POST['pre_approve'];
```

Comment:
oups! that should be `(int)$_POST['pre_approve']`.

Reply:
Good catch! Fixed